main.py

```diff
@@ -1,12 +1,346 @@
+            response = customer.update_customer(
+                name=name, postal_code=postal_code, phone=phone)
+            customer_info = customer.get_customer_by_id(
+                name=name, id=response["id"])
+            print(
+                f"Customer {name} was successfully updated: \n{customer_info}")
+            customer.selected_customer = None
+
+        elif choice == '8':
+            print('''
+            [ DELETE A CUSTOMER ]
+            ''')
+            cancelled_customer = select_customer(customer)
+            if cancelled_customer is None:
+                # break
+                continue
+            print(
+                f"\n{cancelled_customer['name']} found out about Netflix, huh?")
+            customer.delete_customer()
+            print("Customer successfully deleted. I may lose my job at this rate.")
+
+        elif choice == '9':
+            print('''
+            [ GET INFORMATION ON ONE CUSTOMER ]
+            ''')
+            select_customer(customer)
+
+        elif choice == '10':
+            print('''
+            [ GET INFORMATION ON ALL CUSTOMERS ]
+            ''')
+            for each_customer in customer.list_customers():
+                print(each_customer)
+
+#=====================================================#
+#                  RENTAL CHOICES                     #
+#=====================================================#
+
+        elif choice == '11':
+            print('''
+            [ CHECK OUT A VIDEO TO A CUSTOMER ]
+            ''')
+            video_id = input("  > Enter the video id: ")
+            # check if it is a valid id
+            if video_id.isnumeric():
+                video_id = int(video_id)
+                video.get_video(id=video_id)
+
+            if video.selected_video:
+                print(f"Selected video: \n{video.selected_video}")
+            else:
+                print("Video not found. Please enter a valid id.")
+                continue
+
+            video.selected_video = None
+            customer_id = input("  > Enter the customer id: ")
+            # check if valid id
+            if customer_id.isnumeric():
+                customer_id = int(customer_id)
+                customer.get_customer_by_id(id=customer_id)
+
+            if customer.selected_customer:
+                print(f"Selected customer: \n{customer.selected_customer}")
+            else:
+                print("Customer not found. Please enter a valid id.")
+                continue
+            customer.selected_customer = None
+
+            print("")
+            checked_out = rental.check_out_video(
+                customer_id=customer_id, video_id=video_id)
+            customer_info = customer.get_customer_by_id(id=customer_id)
+            video_info = video.get_video(id=video_id)
+            print(f"Rental info: \n{checked_out}")
+            print(
+                f"{video_info['title']} successfully checked out to {customer_info['name']}!")
```

Comment:
Future enhancement: adding logic to handle what happens if either customer_id or video_id passed don't exist. Currently, the CLI will crash when this happens.